rackcity/views/decommission_views.py

```diff
@@ -87,3 +92,100 @@ def decommission_asset(request):
             },
             status=HTTPStatus.OK
         )
+
+
+@api_view(['POST'])
+@permission_classes([IsAuthenticated])
+def decommissioned_asset_many(request):
+    """
+    List many decommissioned assets. If page is not specified as a query parameter, all
+    assets are returned. If page is specified as a query parameter, page
+    size must also be specified, and a page of assets will be returned.
+    """
```

Comment:
The way this is written definitely takes on some technical debt - this view is almost exactly the same as `asset_many`, and shares a lot of code for paging, sorting, and filtering with some other get-many endpoints.  I personally don't think it's worth refactoring as I think that would cost more time now than the repeated code will cost us next evolution, but I'm open to dissent

Reply:
Sike Julia just refactored this

Reply:
Tagging WIP until that's merged and I can change this to reflect it